packages/mobile-app/data/facades/wallet/oreowalletHandlers.ts

```diff
@@ -0,0 +1,382 @@
+import { f } from "data-facade";
+import {
+  Account,
+  AccountSettings,
+  Output,
+  Transaction,
+  TransactionType,
+  WalletHandlers,
+  WalletStatus,
+} from "./types";
+import { oreoWallet } from "../../wallet/oreowalletWallet";
+import { IRON_ASSET_ID_HEX, Network } from "../../constants";
+import * as Uint8ArrayUtils from "../../../utils/uint8Array";
+
+import {
+  AccountFormat,
+  LanguageKey,
+  LanguageUtils,
+  TransactionStatus,
+} from "@ironfish/sdk";
+import { OreowalletServerApi } from "../../oreowalletServerApi/oreowalletServerApi";
+
+export const walletHandlers = f.facade<WalletHandlers>({
+  createAccount: f.handler.mutation(
+    async ({ name }: { name: string }): Promise<Account> => {
+      const account = await oreoWallet.createAccount(Network.MAINNET, name);
```

Comment:
Do we want all of these to be hardcoded to mainnet at the moment?

Reply:
Yep. I still need to build network switching, and our current testnet server doesn't reliably sync up new accounts (Mat's working on getting some fixes in for this)